src/NuGet.Core/NuGet.Build.Tasks/RestoreSettingsUtils.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
is this removing the BOM? Adding it?

Reply:
My IDE keeps adding the BOM if missing, should I undo that?

Reply:
We have a .editorconfig setting that tells it to remove BOM, so as long as you're using something that understands and respects the .editorconfig file (such as VSCode or VS2019), you should be fine.  Pretty much every file before the .editorconfig file set the encoding had the BOM, so first edit of each file will remove it.  Unfortunately project system doesn't seem to respect .editorconfig encoding, so using PM UI to add/remove packages does different things compared to editing the csproj manually, but this file isn't a csproj.